index.html

```diff
@@ -0,0 +1,257 @@
+							<p class="tab-content__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis ornare turpis nec mauris scelerisque, sed suscipit elit dapibus. Aliquam ut eros sed risus convallis venenatis eget vitae mi. Cras dictum est non nunc semper blandit. Quisque convallis tristique dolor eget ultrices. Mauris ac orci quis neque facilisis laoreet. Suspendisse vitae ultrices ipsum, at placerat eros. Proin varius malesuada ultricies. Cras a metus elementum, pellentesque libero eu, faucibus lorem. Integer ac ornare lacus. Ut pretium felis tellus, nec faucibus eros rhoncus eget. Pellentesque ultrices, ligula vestibulum porttitor pharetra, arcu nisl vulputate ligula, id vulputate ligula urna vel tortor. In ullamcorper blandit erat, nec egestas massa commodo consectetur. In nec odio lorem. Donec nulla neque, vehicula in bibendum et, rhoncus sit amet nibh.</p>
+						</div>
+						<div id="tab4" class="tab-content tab-content--4">
+							<h2 class="tab-content__title tab-content__title--4">Text from box name 1</h2>
+							<p class="tab-content__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis ornare turpis nec mauris scelerisque, sed suscipit elit dapibus. Aliquam ut eros sed risus convallis venenatis eget vitae mi. Cras dictum est non nunc semper blandit. Quisque convallis tristique dolor eget ultrices. Mauris ac orci quis neque facilisis laoreet. Suspendisse vitae ultrices ipsum, at placerat eros. Proin varius malesuada ultricies. Cras a metus elementum, pellentesque libero eu, faucibus lorem. Integer ac ornare lacus. Ut pretium felis tellus, nec faucibus eros rhoncus eget. Pellentesque ultrices, ligula vestibulum porttitor pharetra, arcu nisl vulputate ligula, id vulputate ligula urna vel tortor. In ullamcorper blandit erat, nec egestas massa commodo consectetur. In nec odio lorem. Donec nulla neque, vehicula in bibendum et, rhoncus sit amet nibh.</p>
+						</div>
+					</div>
+				</div>
+			</div>
+		</section>
+	</main>
+	<!-- end of main -->
+
+	<!-- footer -->
+	<footer class="footer">
+		<section class="container clearfix">
+			<div class="footer-top">
+				<figure id="section-2" class="gallery js--section">
+					<div class="gallery-box gallery-box--1">
+						<div class="gallery-box__content">
+							<div class="gallery-box__content-octagon">
+								<picture>
+									<source media="(min-width: 1170px)" srcset="images/boy_desktop_1170.jpg 1170w">
+									<source media="(min-width: 1024px)" srcset="images/boy_desktop_1024.jpg 1024w">
+									<source media="(min-width: 768px)" srcset="images/boy_tablet.jpg 768w">
+									<img src="images/boy.jpg" alt="" class="gallery-box__content-octagon__img-move">
+								</picture>
+							</div>
+							<figcaption class="gallery-box__content-thumb">
+								<h3 class="gallery-box__content-thumb__title gallery-box__content-thumb__title--1"><span class="gallery-box__content-thumb__title gallery-box__content-thumb__title--bg">Box Caption</span></h3>
```

Comment:
Przenieś `<span>` do  nowej linijki. Zrób to samo we wszystkich innych przypadkach.